app/models/message.rb

```diff
@@ -23,9 +23,28 @@ class Message < ApplicationRecord
 
   after_create_commit :broadcast_message
   after_create_commit :increment_unread_messages_counter
+  after_create_commit :notify_addressee
 
   private
 
+  def notify_addressee
+    return unless addressee_push_token.any?
+    NotificationService.new.notify(addressee_push_token,
+                                   I18n.t('api.notifications.new_message'), data)
+  end
+
+  def data
```

Comment:
I would change this method name, to be more descriptive than just `data`, but its up to you

Reply:
Agree, this name should be more descriptive. Please change it.